plugins/woocommerce-blocks/assets/js/blocks/mini-cart/block.tsx

```diff
@@ -251,9 +254,15 @@ const MiniCartBlock = ( attributes: Props ): JSX.Element => {
 			<button
 				className={ `wc-block-mini-cart__button ${ colorClassNames }` }
 				onClick={ () => {
-					if ( ! isOpen ) {
-						setIsOpen( true );
-						setSkipSlideIn( false );
+					if ( onCartClickBehaviour === 'open_drawer' ) {
+						if ( ! isOpen ) {
+							setIsOpen( true );
+							setSkipSlideIn( false );
+						}
+					} else if (
+						onCartClickBehaviour === 'navigate_to_checkout'
+					) {
+						window.location.href = CHECKOUT_URL;
```

Comment:
I wonder if instead of if/else and nested if, it would be simpler a more readable to just return early, like this:

```suggestion
                     if ( onCartClickBehaviour === 'navigate_to_checkout' ) {
                        window.location.href = CHECKOUT_URL;
                        return;
                     }
                     
                     if ( ! isOpen ) {
                         setIsOpen( true );
                         setSkipSlideIn( false );
                     }
```

Reply:
Sure! For reasons entirely my own bias, I never liked early returns, but I think in simple cases like this it looks better. Thanks for the suggestion 😄